src/components/Team/TeamList.js

```diff
@@ -2,88 +2,207 @@ var team = [{
-  }]
-},{
-  'hardware' : [{
-    'name' :'Pranjal Paliwal',
-    'github' : 'https://github.com/betterclever',
-    'avatar': 'https://avatars1.githubusercontent.com/u/16309034?v=4&s=460'
-  }]
-},{
-  'bots' :[{
-    'name' : 'Manveer Singh',
+  'server': [{
+    'name': 'Saurabh Jain',
+    'github': 'https://github.com/saurabhjn76',
+    'avatar': 'https://scontent.fmaa2-1.fna.fbcdn.net/v/t1.0-1/c80.0.480.480/284797_228629740506894_1570458_n.jpg?oh=9790e3be67ff793ab29a474b915b3d15&oe=5A0AED30',
+    'designation': 'Java + Web Developer',
+    'twitter': 'https://twitter.com/mariobehling',
+    'linkedin': 'https://www.linkedin.com/in/mariobehling/',
+    'blog': '#'
+  }, {
+    'name': 'Chetan Kaushik',
+    'github': 'https://github.com/dynamitechetan',
+    'avatar': 'https://avatars1.githubusercontent.com/u/14369357?v=4&s=400',
+    'designation': 'Java + Web Developer',
+    'twitter': 'https://twitter.com/DynamiteChetan',
+    'linkedin': 'https://www.linkedin.com/in/chetankaushik/',
+    'blog': '#'
+  }, {
+    'name': 'Dravit Lochan Gupta',
+    'github': 'https://github.com/DravitLochan',
+    'avatar': 'https://avatars0.githubusercontent.com/u/14837478?v=4&s=460',
+    'designation': 'Java + Web Developer',
+    'twitter': 'https://twitter.com/DravitLochan',
+    'linkedin': 'https://www.linkedin.com/in/mariobehling/',
+    'blog': '#'
+  }, {
+    'name': 'Jyothir Aditya',
+    'github': 'https://github.com/jyoth1raditya',
+    'avatar': 'https://avatars2.githubusercontent.com/u/24853337?v=4&s=460',
+    'designation': 'Java + Web Developer',
+    'twitter': 'https://twitter.com/mariobehling',
+    'linkedin': 'https://www.linkedin.com/in/mariobehling/',
+    'blog': '#'
+  },
+  {
+    'name': 'Chirag Wadhera',
+    'github': 'https://github.com/chiragw15',
+    'avatar': 'https://avatars1.githubusercontent.com/u/16978820?v=4&s=460',
+    'designation': 'Android Developer',
+    'twitter': ' https://twitter.com/chiragw15',
+    'linkedin': ' https://www.linkedin.com/in/chirag-wadhera-155b22106/',
+    'blog': 'http://blog.fossasia.org/?author_name=chiragw15'
+  }, {
+    'name': 'Mayank Tripathi',
+    'github': 'https://github.com/mayank408',
+    'avatar': 'https://avatars0.githubusercontent.com/u/22056321?v=4&s=460',
+    'designation': 'Android Developer',
+    'twitter': 'https://twitter.com/mayank_trp408',
+    'linkedin': 'https://www.linkedin.com/in/mayank-tripathi-a49563126/',
+    'blog': 'https://mayank408.github.io/'
+  }, {
+    'name': 'Amit Tiwary',
+    'github': 'https://github.com/amitiwary999',
+    'avatar': 'https://avatars3.githubusercontent.com/u/14003244?v=4&s=460',
+    'designation': 'Android Developer',
+    'twitter': 'https://twitter.com/mariobehling',
+    'linkedin': 'https://www.linkedin.com/in/amit-tiwary-8bb081103',
+    'blog': '#'
+  },
+  {
+    'name': 'Rishi Raj',
+    'github': 'http://github.com/rishiraj824',
+    'avatar': 'https://avatars3.githubusercontent.com/u/11540785?v=4&s=460',
```

Comment:
Images need to be on the repository itself. We should not depend on external services and website.